docs/userguide/parameters.md

```diff
@@ -1,4 +1,204 @@
 # Parameters
 
-This page will soon contain information about parameters.
-In the meantime, please see the [examples](../../examples/examples) instead.
\ No newline at end of file
+Parameters are fundamental for BayBE, as they configure the ```SearchSpace``` and serve 
+as the direct link to the controllable variables in your experiment. 
+Before starting an iterative campaign, the user is required to specify the exact 
+parameters they can control and want to consider in their optimization.
+
+```{note}
+BayBE identifies each parameter by a ```name```. All parameter names in one 
+campaign must be unique.
+```
+
+Because under the hood they need to be treated very differently, BayBE distinguishes 
+two parameter types: Discrete and continuous parameters.
+
+## Continuous Parameters
+
+### ```NumericalContinuousParameter```
+This is currently the only continuous parameter BayBE supports. 
+This parameters type defines possible values from a numerical interval called 
+```bounds```, and thus has an infinite amount of possibilities. 
+Unless restrained by constraints, BayBE will consider any possible parameter value that 
+lies within the chosen interval.
+
+```python
+from baybe.parameters import NumericalContinuousParameter
+
+NumericalContinuousParameter(
+   name = "Temperature",
+   bounds = (0, 100),
+)
+```
+
+## Discrete Parameters
+A discrete parameter has a finite set of possible values. 
+These values can be numeric or label-like and are transformed internally before being 
```

Comment:
are transformed: strictly speaking, not true for all, e.g. the numericaldiscrete one. Perhaps add an "optionally"? Perhaps you find a better solution